dist/debian/debian/scylla-server.postrm

```diff
@@ -6,8 +6,14 @@ case "$1" in
     purge|remove)
         rm -rf /etc/systemd/system/scylla-housekeeping-daily.service.d/
         rm -rf /etc/systemd/system/scylla-housekeeping-restart.service.d/
-        rm -rf /etc/systemd/system/scylla-server.service.d/
         rm -rf /etc/systemd/system/scylla-helper.slice.d/
+        # We need to keep dependencies.conf and sysconfdir.conf on 'remove',
+        # otherwise it will be missing after rollback.
+        if [ "$1" = "purge" ]; then
+            rm -rf /etc/systemd/system/scylla-server.service.d/
+        else
+            rm -f /etc/systemd/system/scylla-server.service.d/capabilities.conf
```

Comment:
Why remove this? shouldn't we keep it, since nothing will rebuild it?

Reply:
It will re-install by scylla_post_install.sh since it is not part of the package, but it won't be problem we leave the file, and 'apt-get remove' defined as leave conffiles unlike 'apt-get purge'.
So I deleted the line and pushed new version:
https://github.com/scylladb/scylla/pull/5748/files

BTW, I found another problem. I don't see capalibities.conf installed on Ubuntu, seems like scylla_post_install.sh issue: #5764